go/client/cmd_paperprovision.go

```diff
@@ -23,16 +23,23 @@ func NewCmdPaperProvision(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cl
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(NewCmdPaperProvisionRunner(g), "paperprovision", c)
 		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "keep-paper-key, k",
```

Comment:
Yeah I'd have done it that way except that `paperprovision` is only used in `kbfsdocker`.